apps/web/components/booking/BookingListItem.tsx

```diff
@@ -654,45 +672,255 @@ const FirstAttendee = ({
 type AttendeeProps = {
   name?: string;
   email: string;
+  id: number;
+  noShow: boolean;
+};
+
+type NoShowProps = {
+  bookingUid: string;
+  isPast: boolean;
 };
 
-const Attendee = ({ email, name }: AttendeeProps) => {
+const Attendee = (attendeeProps: AttendeeProps & NoShowProps) => {
+  const { email, name, bookingUid, isPast, noShow: noShowAttendee } = attendeeProps;
+  const { t } = useLocale();
+
+  const [noShow, setNoShow] = useState(noShowAttendee);
+  const [openDropdown, setOpenDropdown] = useState(false);
+
+  const noShowMutation = trpc.viewer.public.noShow.useMutation({
+    onSuccess: async () => {
+      noShow
+        ? showToast(
+            t("x_marked_as_no_show", {
+              x: email,
+            }),
+            "success"
+          )
+        : showToast(
+            t("x_unmarked_as_no_show", {
+              x: email,
+            }),
+            "success"
+          );
+    },
+    onError: (err) => {
+      showToast(err.message, "error");
+    },
+  });
+
+  function toggleNoShow({
+    attendee,
+    bookingUid,
+  }: {
+    attendee: { email: string; noShow: boolean };
+    bookingUid: string;
+  }) {
+    noShowMutation.mutate({ bookingUid, attendees: [attendee] });
+    setNoShow(!noShow);
+  }
+
   return (
-    <a className="hover:text-blue-500" href={`mailto:${email}`} onClick={(e) => e.stopPropagation()}>
-      {name || email}
-    </a>
+    <Dropdown open={openDropdown} onOpenChange={setOpenDropdown}>
+      <DropdownMenuTrigger asChild>
+        <button
+          onClick={(e) => e.stopPropagation()}
+          className="radix-state-open:text-blue-500 hover:text-blue-500">
+          {noShow ? (
+            <s>
+              {name || email} <Icon name="eye-off" className="inline h-4" />
+            </s>
+          ) : (
+            <>{name || email}</>
+          )}
+        </button>
```

Comment:
NIT: 

Does this need to be a button? I think without the "asChild" on the trigger this is not needed.

Reply:
Apparently it is adding unnecessary padding that I couldn't get rid of for some reason. I think we can do a follow up here if needed.